include/results.h

```diff
@@ -212,4 +213,7 @@
 /* virus */
 #define REMEDY_VIRUS _("ClamAV has found a virus in the named file.  This may be a false positive, but you should manually inspect the file in question to ensure it is clean.  This may be a problem with the ClamAV database or detection.  If you are sure the file in question is clean, please file a bug with rpminspect for further help.")
 
+/* politics */
+#define REMEDY_POLITICS _("A file with potential politically sensitive content was found in the package.  If this file is permitted, it should added to the rpminspect vendor data package for the product.")
```

Comment:
Small nit:

"If this file is permitted, it should added to the rpminspect vendor data package"

to

"If this file is permitted, it should be added to the rpminspect vendor data package"

Reply:
Fixed and pushed.